api/rpc/account/account.proto

```diff
@@ -171,7 +173,7 @@ message AccountRequest {
   string name = 1;
 }
 
-message Account {
+message AccountInfo {
```

Comment:
I don't think you need to rename this to AccountInfo. For one, I don't think the data resources should be in the rpc package bundled with the service api resources, but all in another package under one flat namespace (like `amp/schema`, so when imported, looks like schema.Account, not to be confused with rpc.Account).